src/move_embargo_args.py

```diff
@@ -82,8 +83,6 @@ def parse_args():
     # If move is true, then you'll need write
     # permissions from the fromrepo (embargo)
     butler = Butler(namespace.fromrepo, writeable=namespace.move)
-    print("temp from path:", namespace.fromrepo)
-    print("temp to path:", namespace.torepo)
     registry = butler.registry
     dest = Butler(namespace.torepo, writeable=True)
     scratch_registry = dest.registry
```

Comment:
`scratch` may not be a great name, as it's actually the permanent archive :)

Reply:
Renaming scratch --> prompt